documents/coredocument.go

```diff
@@ -76,6 +76,11 @@ type CoreDocument struct {
 	Document coredocumentpb.CoreDocument
 }
 
+type Collaborators struct {
```

Comment:
```suggestion
type CollaboratorsAccess struct {
```
At the moment is very simple as we do not care about field access and that is ok.

Reply:
:+1: